server/controllers/importers/sif.js

```diff
@@ -37,6 +81,15 @@ module.exports = function (sif) {
   });
 
   var networkType = sifNetworkType(entries);
+  if (networkType.warnings !== null) {
+    warnings.push(networkType.warnings);
+  }
+
+  if (networkType.errors != null) {
+    for (i = 0; i < networkType.errors.length; i++) {
+      errors.push(networkType.errors[i]);
+    }
```

Comment:
This block seems rewritable as a `forEach` call:

    networkType.errors.forEach(function (error) {
        errors.push(error);
    });

It’s a tiny bit more compact.